thirdparty/github.go

```diff
@@ -47,6 +61,8 @@ var UnblockedGithubStatuses = []string{
 	githubPRUnstable,
 }
 
+var tracer = otel.GetTracerProvider().Tracer(packageName)
```

Comment:
I would move the otel-related global variables to their own file.

Reply:
done